spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -470,10 +475,27 @@ private void activateOptions() {
 			this.connectionFactory.setUsername(this.username);
 			this.connectionFactory.setPassword(this.password);
 			this.connectionFactory.setVirtualHost(this.virtualHost);
+			if (this.clientConnectionProperties != null) {
+				updateClientConnectionProperties();
+			}
 			setUpExchangeDeclaration();
 			this.senderPool = Executors.newCachedThreadPool();
 		}
 
+		private void updateClientConnectionProperties() {
+			String[] props = this.clientConnectionProperties.split(",");
```

Comment:
And why we can't use the same `prop:name[,prop:name]...` approach for all other our appenders?
Yes, leave the extension good hook, but also allow to configure those appenders the same coma-separated value way.
WDYT?